taworks/taform/templates/taform/base.html

```diff
@@ -2,6 +2,7 @@
 <link rel="shortcut icon" href="{{ STATIC_URL }}/static/favicon/favicon.ico">
 <header id = "banner"></header>
 <head>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
```

Comment:
can we do @import url(fonts.css);
in our style.css similar to how admin currently does it

check
static/admin/css/base.css
and
static/admin/css/fonts.css